mod/layout/set/abstract/thumbnail.rb

```diff
@@ -1,20 +1,20 @@
 format :html do
-  view :thumbnail_plain do |args|
+  view :thumbnail_plain, cache: :never do |args|
```

Comment:
none of these should be necessary. 

We can't just keep slapping `cache: never` everywhere.  If there's a bug in the view caching we have to fix it.

Reply:
My understanding was that we can't use view caching if we use args.